README.md

```diff
@@ -145,6 +152,117 @@ readData(dataInput) {
 }
 ```
 
+# Using Node.js Client with Hazelcast IMDG
+
+## 1. Node.js Client API Overview
+
+The first step is configuration. You can configure the Node.js client declaratively or programmatically. We will use the programmatic approach throughout this chapter. Please refer to the [Node.js Client Declarative Configuration section](#declarative-configuration) for details.
+
```

Comment:
I think this chapter is a good place to mention that most of the API is promise based and they can use these in async/await fashion too if they like. I saw some questions on gitter that the user calls a lock and they just do not wait for lock call to be resolved. Then they complain that the locks do not work. I think if we can convey the general usage of the api to the user. It will be easier for them to understand everything.